frontend/src/main/pages/UCSBOrganization/UCSBOrganizationCreatePage.js

```diff
@@ -0,0 +1,52 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import UCSBOrganizationForm from "main/components/UCSBOrganization/UCSBOrganizationForm";
+import { Navigate } from 'react-router-dom'
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+
+export default function UCSBOrganizationCreatePage({storybook=false}) {
+  const objectToAxiosParams = (organization) => ({
+    url: "/api/ucsborganization/post",
+    method: "POST",
+    params: {
+     orgCode: organization.orgCode,
+     orgTranslationShort: organization.orgTranslationShort,
+     orgTranslation: organization.orgTranslation,
+     inactive: organization.inactive
+    }
+  });
+
+  const onSuccess = (organization) => {
+    toast(`New organization Created - orgCode: ${organization.orgCode} orgTranslationShort: ${organization.orgTranslationShort}
+    orgTranslation: ${organization.orgTranslation} inactive: ${organization.inactive}
+    `);
+  }
+
+  const mutation = useBackendMutation(
+    objectToAxiosParams,
+     { onSuccess }, 
+     // Stryker disable next-line all : hard to set up test for caching
+     ["/api/ucsborganization/all"] // mutation makes this key stale so that pages relying on it reload
```

Comment:
this should be /uscsborganizations/all with an s at the end because that's what your api url is